cmds/uinit/uinit.go

```diff
@@ -14,27 +14,68 @@ import (
 
 var (
 	commands = []string{
-		"/buildbin/date",
-		"/buildbin/ls -l",
+		"/bin/bash",
```

Comment:
we definitely don't want to default to bash, in the long term, but this is a fine decision for now.